Template.php

```diff
@@ -1158,9 +1158,105 @@ protected function expand_by_doi($force = FALSE) {
     }
   }
   
+  protected function expand_by_jstor() {
+    if ($this->incomplete() === FALSE) return FALSE;
+    if ($this->blank('jstor')) return FALSE;
+    $jstor = $this->get('jstor');
+    if (preg_match("~[^0-9]~", $jstor) === 1) return FALSE ; // Only numbers in stable jstors
```

Comment:
Will $jstor already be trimmed (I believe so?), or could we end up with whitespace at one end or the other?

Reply:
You are right.  Some moms where I live use Vic’s vapor rub for all problems.  I similarly apply trim() to all texts.  I will remove